### PR TITLE
[FEATURE] Build app container with php 7.0 support

### PR DESCRIPTION
**Pull request by violeng**
Please tag with version 1.1.0

### CODE REVIEW

File: config/app/php-cli.ini [hunk 1]
```diff
@@ -1920,26 +1819,60 @@ ldap.max_links = -1
...
 ;curl.cainfo =
 
-[apc]
-apc.enabled = On
-apc.enable_cli = On
-apc.shm_size = 128M
-apc.ttl = 3600
-apc.user_ttl = 7200
-apc.max_file_size = 1M
-apc.stat = 1
```
Comment (visay) [inline config/app/php-cli.ini:1941]: @violeng you don't want apc module enable for wordpress? if yes, it's fine to remove it like that.

File: config/app/php.ini [hunk 2]
```diff
@@ -1920,26 +1819,60 @@ ldap.max_links = -1
...
 ;curl.cainfo =
 
-[apc]
-apc.enabled = On
-apc.enable_cli = On
-apc.shm_size = 128M
-apc.ttl = 3600
-apc.user_ttl = 7200
-apc.max_file_size = 1M
-apc.stat = 1
```
Comment (visay) [inline config/app/php.ini:1941]: @violeng same question here

File: config/app/php-cli.ini (hunk 1 above)
Comment (violeng) [reply config/app/php-cli.ini:1941]: I don't need it.

File: config/app/php.ini (hunk 2 above)
Comment (violeng) [reply config/app/php.ini:1941]: Don't need it, just remove it.

**Pull request merged**